toolkit/scripts/toolchain/cross/gcc_cross.sh

```diff
@@ -17,13 +17,18 @@
 
```

Comment:
`gcc_cross.sh` seems to no longer have execute permissions after this update.

Reply:
I've had to run `chmod +x gcc_cross.sh` prior to running it in order to give it execute permissions. Would having to run that cause problems in the Mariner build?

Reply:
I think it is fine to request the script user to run `chmod +x gcc_cross.sh`. Just noting that the execution permissions changed between the previous script and this PR. Wasn't sure if it was an intentional change or not.

To add more to the confusion, I think my PR added execute permissions to `gcc_cross.sh`

Reply:
Hmm that's strange. It definitely wasn't an intentional change. I've had times where I needed to re-run `chmod` if I was overwriting the script with another copy. I'm going to add a comment at the top of the script to call out that `chmod` might be required prior to running the script.